dedicated-community-leader.md

```diff
@@ -0,0 +1,80 @@
+## Title
+
+_Dedicated Community Leader_
+
+## Problem
+
+When starting an InnerSource initiative it is crucial to nominate the right people to lead the communities. Selecting the wrong persons and/or not providing enough capacity for them risks wasted effort and ultimately the failure of the InnerSource initiative.
```

Comment:
@NewMexicoKid : "How do you ensure that the new InnerSource initiative has the right leadership for the communities?"